tests/installation/partitioning_expert.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: 
+# Maintainer: Christopher Hofmann <cwh@suse.de> 
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    send_key $cmd{expertpartitioner};
+    assert_screen 'expert-partitioner';
+
+    assert_and_click 'hard-disks';
+    assert_and_click 'home';
+
+    send_key 'alt-d'; # Delete
+    send_key 'alt-y'; # Confirm with 'yes'
+    assert_and_click 'hard-disks';
+    save_screenshot;
+
+    send_key $cmd{accept};
+    die "/home still there" if check_screen('home');
```

Comment:
Please don't use check_screen with default timeout. Either surround the previous `send_key` with `wait_screen_change` or use a multi-tag assert_screen plus `match_has_tag`